controllers/search/search.go

```diff
@@ -64,6 +64,10 @@ func SearchHandler(c *gin.Context) {
 	if c.Query("order") == "true" {
 		searchForm.SortOrder = true
 	}
+	searchForm.SortType, err = strconv.Atoi(c.Query("sort"))
```

Comment:
Move this to https://github.com/NyaaPantsu/nyaa/pull/1371/files#diff-2ff0b4807eadb9a31e1e7b187f5da802R50
It's better to have it directly set in the NewSearchForm function